src/DI/TranslationExtension.php

```diff
@@ -141,8 +142,11 @@ public function loadConfiguration()
 		$builder->addDefinition($this->prefix('selector'))
 			->setClass(MessageSelector::class);
 
+		$builder->addDefinition($this->prefix('intlFormatter'))
+			->setClass(IntlFormatter::class);
```

Comment:
Also for compatibility with older symfony, add an `interface_exists` check.

Reply:
Yes, it will throw an error. It looks (in the symfony/translation sources) like this interface is not directly used, only through the `MessageFormatter` proxy,

Reply:
To the autowiring: I could only achieve this (autowiring `IntlFormatter`) if I disable the autowiring of the `MessageFormatter` service, but that is used in the `Translator` service, so I cannot do that..

Reply:
Yes, you're right, this works.. But I had to change the `MessageFormatter` in `Translator` to `MessageFormatterInterface` for this to work, I'm not sure if the tests will pass..

Reply:
And this can be a breaking change if someone changed the formatter service in nette through `class:` instead of 'factory:'..